modules/ossm-config-sampling.adoc

```diff
@@ -5,7 +5,7 @@
 [id="ossm-config-sampling_{context}"]
 = Adjusting the sampling rate
 
-The distributed tracing sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces.
+The Envoy proxy sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces.
```

Comment:
Looks good, my only comment is that it feels a little bit repetitive to say " A high sampling rate consumes cluster resources and performance but is useful when debugging issues." at line 8 and "Setting the value to `10000` is useful for debugging, but can affect performance." at line 19.

Reply:
Julie and I discussed and made minor edits.